src/vmware/pvscsi.c

```diff
@@ -138,7 +138,9 @@ static void pvscsi_action_io_queued(pvscsi dev, struct pvscsi_hcb *hcb, u16 targ
     spin_unlock(&dev->queue_lock);
```

Comment:
I noticed that pvscsi_rx_service_bh holds dev->queue_lock while processing the list, which includes calling pvscsi_action_io. Would it better to move this unlock below the pvscsi_action_io here instead of unlocking and re-locking?

Reply:
I agree - update pushed.